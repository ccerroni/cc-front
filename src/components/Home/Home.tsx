export const Home: React.FC = () => {
  return (
    <div className="m-8">
      <h1 className="text-3xl">Home Page</h1>
    </div>
  )
}