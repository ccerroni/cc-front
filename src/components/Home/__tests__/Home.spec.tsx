import { render } from '@testing-library/react';
import { Home } from '../Home';

describe('test suites for Home', () => {

  it('test', () => {
    expect(true).toBeTruthy();
  });
  
  it('should render the component correctly', () => {
    // const { container } = render(<Home />);
    // expect(container).toMatchSnapshot();

    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
  });
});
