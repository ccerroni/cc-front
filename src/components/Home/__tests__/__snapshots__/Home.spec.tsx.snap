// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test suites for Home should render the component correctly 1`] = `
<DocumentFragment>
  <div
    class="m-8"
  >
    <h1
      class="text-3xl"
    >
      Home Page
    </h1>
  </div>
</DocumentFragment>
`;
